OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -621,6 +628,14 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
     event->SetOriginFileDescriptor(fd);
     DeferEvent(std::move(event));
     ++stats_.sample_count;
+
+  } else {
+    // Skip PERF_RECORD_SAMPLEs with an unknown size. These are normally
+    // time-based samples with abi == PERF_SAMPLE_REGS_ABI_NONE and no
+    // registers, and with size == 0 and no stack. Usually, these samples have
+    // pid == tid == 0, but that's not always the case, for example when a
+    // process exits while tracing.
```

Comment:
It is not clear from this comment what happens to size/abi/pid/tid when process exits... Do we get non-zero pid/tid but zero size? Please clarify the comment here.

Reply:
also what happens for gpu_events?

Reply:
I tried to clarify, please have another look.